portal-web/test/functional/com/liferay/portalweb/macros/FormFields.macro

```diff
@@ -978,6 +965,27 @@ definition {
 			locator1 = "Form#SIDEBAR_ADVANCED_PREDEFINED_VALUE");
 	}
 
+	macro toggleCKEditorContentStyle {
+		if (isSet(content)) {
+			FormFields.editTextBodyToolBar(content = "${content}");
+		}
```

Comment:
@gleicelisbino, can we keep the macro `toggleCKEditorContentStyle` only in charge of toggle the content styles?

In the test, we invoke the macro `editTextBodyToolBar` to insert the content `Test`, then we invoke the macro `toggleCKEditorContentStyle` to enable and disable the content styles every time it's needed. :wink:

Reply:
@natocesarrego I tried to do this, but it didn't work.